docs/_blog/getting_started_with_rust_hwi.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Getting Started with rust-hwi"
+description: "This post will help one understand and develop for hardware wallets using BDK"
+authors:
+    - Wszdexdrf
+date: "2022-08-16"
+tags: ["BDK", "Development", "Hardware Wallets"]
+hidden: true
+draft: false
+---
+# Getting started with rust-hwi
```

Comment:
It seems that this title isn't necessary as the deployment takes the title from the metatags above.
You can see they are duplicated in the blog preview.

Reply:
Thanks for pointing that out.